code/game/gamemodes/cult/runes.dm

```diff
@@ -535,7 +535,7 @@ var/list/teleport_runes = list()
 	if(rune_in_use)
 		return
 	for(var/mob/living/M in orange(1,T))
-		if(M.stat == DEAD)
+		if(M.stat == DEAD && !iscultist(M))
```

Comment:
Also if they're using monkeys from, like, genetics it's slightly moot because they could humanize them; it's good to reward ingenuity, just not all ingenuity.